src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -120,7 +120,10 @@ class DiskImageEditor extends Component {
           bootable: disk.get('bootable'),
           size: 0,
           storageDomain: disk.get('storageDomainId'),
-          format: disk.get('format'),
+
+          // NOTE: webadmin always uses raw when creating disks.  We need to look at
+          //       the sparse flag to determine the proper format to display.
+          format: disk.get('sparse') ? 'cow' : 'raw',
```

Comment:
I think it's better to change the field name to "type" or "sparse" instead of "format" and the values to "prealocated"/"thin provision" or boolean accordingly. 
Leaving the of current "format", "cow","raw" values is wrong because it refers to a different field.

Reply:
I renamed the form state field to `diskFormat` to match the label on the form.